tokio-console/src/view/mod.rs

```diff
@@ -168,32 +177,56 @@ impl View {
         update_kind
     }
 
+    fn handl_help_popup(&mut self, event: crossterm::event::Event) {
```

Comment:
typo: should probably be
```suggestion
    fn handle_help_popup(&mut self, event: crossterm::event::Event) {
```